.github/workflow-template/jobs/compute-node-test.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         run: |
           cargo doc --document-private-items --no-deps
 
+      - name: Run rust failpoints test
```

Comment:
Shall we enable clippy check for failpoints? (cargo clippy should be run with --all-features)

Reply:
good point. added.